web/beacon-app/cypress/e2e/ProjectsPage.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Project Page
+
+I want to navigate the Project page
+
+Scenario: Navigating the Project Page
+
+Given I'm logged into Beacon
+And I click Projects
+Then I navigate to the project page
+Then I should see the quick view data
+And I should see the project list table
+When I click on the Create Project button
+Then I should see the create project modal
+#When I click the close button
```

Comment:
I think we need to add an id to the close button in the design system as I kept getting a Cypress error stating that `text-area` was covering the `button`. I've left this here and can update the test in the future.